### PR TITLE
new issue template

### PR DESCRIPTION
**Pull request by keyvanelhami**
### This PR:

Added: New issue template when Balsamic fails to be able to troubleshoot

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/failed_analysis.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+labels:
+assignees:
+
+---
+
+**Error message**
+Copy either the error message from the e-mail or copy the line(s) in the error log showing the error message(s).
+
+**Link to the error message**
+Copy the path for the error file in Hasta which includes the error message(s).
```
Comment (hassanfa) [inline .github/ISSUE_TEMPLATE/failed_analysis.md:15]:
```suggestion
Copy the path for the error file in Hasta which includes the error message(s). In order to get path to the file, run the following command on Hasta: `ejobinfoprint jobid` (replace job id with actual job id).
```

File: .github/ISSUE_TEMPLATE/failed_analysis.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+about: Troubleshooting errors in Balsamic
+case-id:
+lims-id:
+labels:
+assignees:
+
+---
+
+**Error message**
+Copy either the error message from the e-mail or copy the line(s) in the error log showing the error message(s).
```
Comment (hassanfa) [inline .github/ISSUE_TEMPLATE/failed_analysis.md:12]:
```suggestion
Copy either the error message from the e-mail or copy **only the relevant** line(s) in the error log showing the error message(s). Please do not paste the whole error text.
```

File: .github/ISSUE_TEMPLATE/failed_analysis.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+name: Report failed analysis
+about: Troubleshooting errors in Balsamic
+case-id:
+lims-id:
+labels:
+assignees:
+
+---
+
```
Comment (hassanfa) [inline .github/ISSUE_TEMPLATE/failed_analysis.md:10]:
```suggestion
**Case or sample information**
Internal sample id or case id: (do not put LIMS ids here, only anonymized ids are allowed)
Amount read-pairs:
Sequence type (WES, TGA, WGS):
```

Comment (keyvanelhami) [reply .github/ISSUE_TEMPLATE/failed_analysis.md:10]: The LIMS ID us anynomized and should be added in the issue template

**Comment by keyvanelhami** [comment]
@hassanfa I made some changes according to your comments

**Comment by hassanfa** [comment]
Strange, I can't find the final changes. Could you double check if you've pushed them?

**Comment by keyvanelhami** [comment]
I forgot to first pull before pushing. You can see the changes now

File: .github/ISSUE_TEMPLATE/failed_analysis.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+case-id:
+lims-id:
+Amount read-pairs:
+Sequence type (WES, TGA, WGS):
+
+---
+**Case or sample information**
+Internal sample id or case id: (only anonymized ids are allowed)
+Amount read-pairs:
+Sequence type (WES, TGA, WGS):
```
Comment (hassanfa) [inline .github/ISSUE_TEMPLATE/failed_analysis.md:13]:
```suggestion
Sequence type (WES, TGA, WGS):

```

File: .github/ISSUE_TEMPLATE/failed_analysis.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+---
+name: Report failed analysis
+about: Troubleshooting errors in Balsamic
+case-id:
+lims-id:
+Amount read-pairs:
+Sequence type (WES, TGA, WGS):
```
Comment (hassanfa) [inline .github/ISSUE_TEMPLATE/failed_analysis.md:7]:
```suggestion
```

**Comment by hassanfa** [comment]
👍  merging it.

**Pull request merged**